airflow/providers/google/leveldb/operators/leveldb.py

```diff
@@ -0,0 +1,79 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from typing import Any, Dict, List, Optional
+
+from airflow.models import BaseOperator
+from airflow.providers.google.leveldb.hooks.leveldb import LevelDBHook
+from airflow.utils.decorators import apply_defaults
+
+
+class LevelDBOperator(BaseOperator):
+    """
+    Execute command in LevelDB
+
+    .. seealso::
+        For more information on how to use this operator, take a look at the guide:
+        :ref:`howto/operator:LevelDBOperator`
+    """
+
+    @apply_defaults
+    def __init__(
+        self,
+        *,
+        command: str,
+        key: bytes,
+        value: bytes = None,
+        keys: List[bytes] = None,
+        values: List[bytes] = None,
+        leveldb_conn_id: str = 'leveldb_default',
+        name: str = '/tmp/testdb/',
+        create_if_missing: bool = False,
```

Comment:
```suggestion
        create_if_missing: bool = True,
```
Overall, the PR is Ok by me. Another thing is system test. There's a rule to have system tests in all google providers.
I know @mik-laj, won't be happy if there's no system test for this provider.